src/platforms/common/configuration/releases.mdx

```diff
@@ -7,6 +7,16 @@ notSupported:
   - javascript.electron
 ---
 
+<PlatformSection supported={["unity"]}>
+
+<Note>
+
+The Crash Free rate only indicates real crashes with [native support](platforms/unity/configuration/native-support/) enabled and is otherwise marked as Abnormal. This feature is currently limited to iOS.
```

Comment:
I think it got confusing at the end. With the period then `This feature is currently limited to iOS.`: The whole feature is limited? Or the native crash support? Maybe we leave the specifics out of this sentence. We link to the native support so users can go there to learn more.

Maybe something like:
```suggestion
The Crash Free rate requires [native support](platforms/unity/configuration/native-support/). In platforms that don't have native support, Sentry reports sessions that didn't end gracefully as _Abnormal_.
```

Not sure that's much better.

Maybe @imatwawana can help us out here. 👋 

We're trying to tell users that "Release & Health" work for all platforms you might target your Unity game to. But "Crash Free" stats will be 100% (misleadingly) in all platforms we don't have *native support* which is currently iOS and soon Android.

That means that if you're building a Unity game to on a Play Station 4, Sentry will tell you `0 crashes`, so 100% crash free, but any native crash that might have happened actually resulted in a `Abnormal` session termination, so users should be aware of that and look at those numbers.

For example, from [our own Unity game](https://github.com/getsentry/sentry-defenses) which still doesn't have the native iOS support:

![image](https://user-images.githubusercontent.com/1633368/134252329-45dc7d04-1403-4e38-b63c-82cb2991ade4.png)


Maybe we need a short note here which links to a section further down that explains this in more detail. Looking at my explanation above, it doesn't look like something we can document with a single line.